wdl/model/draft2/src/main/scala/wdl/draft2/model/WdlSyntaxErrorFormatter.scala

```diff
@@ -9,6 +9,12 @@ import scala.collection.JavaConverters._
 
 case class WdlSyntaxErrorFormatter(terminalMap: Map[Terminal, WorkflowSource]) extends SyntaxErrorFormatter {
 
+  // Hashing the terminalMap is very expensive because we re-hash the entire workflow once for every terminal in it (exponential with size of workflow)
+  // To approximate, just use Java's case-class-ignorant implementation (hash functions are allowed to be imperfect)
+  override def hashCode(): Int = System.identityHashCode(this)
+
+  override def equals(obj: Any): Boolean = this eq obj.asInstanceOf[AnyRef]
```

Comment:
Was the `.asInstanceOf[AnyRef]` necessary? It seems a tiny bit dangerous in case it gets passed a `null` (I can't see *why* it ever would, but just in case?)

Reply:
Necessary because one cannot compare `Any` to `AnyRef`; `null.asInstanceOf[AnyRef]` seems to just politely return `null`